src/test/java/steps/BookingSteps.java

```diff
@@ -0,0 +1,73 @@
+package steps;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+
+import java.util.concurrent.TimeUnit;
+
+public class BookingSteps {
+    WebDriver driver;
+    @Before
+    public void setUp(){
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
```

Comment:
You're using raw drivermanager though you added driver factory. So. If you added factory please connect it with tests